docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -802,7 +802,48 @@ created before the partitioning change.
 The connector supports the following commands for use with {ref}`ALTER TABLE
 EXECUTE <alter-table-execute>`.
 
-```{include} optimize.fragment
+##### optimize
+
+The `optimize` command is used for rewriting the content of the specified
+table so that it is merged into fewer but larger files. If the table is
+partitioned, the data compaction acts separately on each partition selected for
+optimization. This operation improves read performance.
+
+All files with a size below the optional `file_size_threshold` parameter
+(default value for the threshold is `100MB`) are merged in case any of the
+following conditions per partition / table (for unpartitioned tables)
```

Comment:
`following conditions are met per partition`
unpartitioned case need not be mentioned, it can be understood as single partition